xml/mysql_banners.xml

```diff
@@ -0,0 +1,850 @@
+    <description>Oracle MySQL - Error: Unable to resolve client hostname</description>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+  </fingerprint>
+
+  <!-- Standard Banner Matches -->
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3}(?:[.-]\d{1,2})?)(?:-m\d{1,2})?(?:-rc)?(?:-alpha)?(?:-beta)?(?:-gamma)?(?:-?[Mm]ax)?(?:-rs)?(?:-modified)?(?:-debug)?(?:-log)?$">
+    <example service.version="4.1.20">4.1.20</example>
+    <example service.version="5.5.23-55">5.5.23-55</example>
+    <example service.version="5.5.18.1">5.5.18.1-log</example>
+    <example service.version="5.7.4">5.7.4-m14</example>
+    <example service.version="5.0.95">5.0.95-rs</example>
+    <example service.version="5.1.65">5.1.65-log</example>
+    <example service.version="5.4.1">5.4.1-beta-log</example>
+    <example service.version="4.1.13">4.1.13-Max-log</example>
+    <example service.version="5.1.33">5.1.33max-log</example>
+    <example service.version="5.0.87">5.0.87-modified-log</example>
+    <example service.version="5.5.3">5.5.3-m3-log</example>
+    <example service.version="4.0.20a">4.0.20a-debug</example>
+    <example service.version="5.1.37">5.1.37-debug-log</example>
+    <description>Oracle MySQL (common)</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3})(?:-alpha)?(?:-beta)?(?:-rc)?-community(?:-max)?-nt(?:-log)?(?:-debug)?(?:-log)?$">
+    <example service.version="4.1.22">4.1.22-community-max-nt</example>
+    <example service.version="5.0.88">5.0.88-community-nt</example>
+    <example service.version="5.0.67">5.0.67-community-nt-log</example>
+    <example service.version="5.1.20">5.1.20-beta-community-nt-debug</example>
+    <example service.version="6.0.0">6.0.0-alpha-community-nt-debug</example>
+    <example service.version="6.0.2">6.0.2-alpha-community-nt-debug-log</example>
+    <description>Oracle MySQL Community Edition with Named Pipes (Windows)</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+    <param pos="0" name="service.edition" value="Community Edition"/>
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.device" value="General"/>
+    <param pos="0" name="os.family" value="Windows"/>
+    <param pos="0" name="os.product" value="Windows"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3})(?:-m\d)?(?:-rc)?(?:-alpha)?(?:-beta)?-[Cc]ommunity(?:-max)?(?:-debug)?(?:-log)?(?:-debug)?$">
```

Comment:
Here you are searching for Community or community, but in the previous one you are just looking for community.  IMO, make both of these (and perhaps all of them) case insensitive